### PR TITLE
docs: last mile helm chart changes

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: docs/content/installation/gateway/kubernetes/helm_advanced.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Helm post render works with stdin/stdout, and kustomize works with files. Let's bridge that gap
+with a shell script:
+
+```bash
+cat > kustomize.sh <<EOF
+#!/bin/sh
+cat > base.yaml
+# you can also use "kustomize build ." if you have it installed.
+exec kubectl kustomize
```
Comment (kdorosh) [inline docs/content/installation/gateway/kubernetes/helm_advanced.md:51]: this requires kubectl 1.14+ right? we should document that

Comment (yuval-k) [reply docs/content/installation/gateway/kubernetes/helm_advanced.md:60]: done

File: docs/content/installation/gateway/kubernetes/helm_advanced.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+## Apply
+
+You can use this command to install \ upgrade your release:
+
+```bash
+kubectl create ns gloo-system
+helm upgrade -i gloo gloo/gloo --namespace gloo-system --post-renderer ./kustomize.sh
+```
+
+Examine the `gateway-proxy` deploymet, you will see the new value:
```
Comment (kdorosh) [inline docs/content/installation/gateway/kubernetes/helm_advanced.md:114]: typo
```suggestion
Examine the `gateway-proxy` deployment, you will see the new value:
```

**Pull request merged**